### PR TITLE
sys-process/nvtop: new package (from ::guru)

### PR DESCRIPTION
**Pull request by vowstar**
NVTOP stands for Neat Videocard TOP, a (h)top like task monitor for GPUs and accelerators. It can handle multiple GPUs and print information about them in a htop-familiar way.

This tool already exists on most distributions and is widely used.

https://repology.org/project/nvtop/versions

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by vowstar** [comment]
@clan

File: sys-process/nvtop/nvtop-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_PREFIX="${EPREFIX}/usr"
```
Comment (thesamesam) [inline sys-process/nvtop/nvtop-9999.ebuild:40]: Hm, do you need this? Shouldn't cmake.eclass set this for us?

File: sys-process/nvtop/nvtop-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		-DCURSES_NEED_WIDE=$(usex unicode)
```
Comment (thesamesam) [inline sys-process/nvtop/nvtop-9999.ebuild:41]: The ncurses dep should really be "[unicode(+)?]" then.

Comment (thesamesam) [reply sys-process/nvtop/nvtop-9999.ebuild:41]: (+) because we may remove it and make it always-on in future

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: sys-process/nvtop/nvtop-9999.ebuild (hunk 2 above)
Comment (vowstar) [reply sys-process/nvtop/nvtop-9999.ebuild:41]: I'll update the dependency to [unicode(+)?] to align with this approach. Thanks for the clarification!

```diff
- sys-libs/ncurses:0=
+ sys-libs/ncurses[unicode(+)?]
```

File: sys-process/nvtop/nvtop-9999.ebuild (hunk 1 above)
Comment (vowstar) [reply sys-process/nvtop/nvtop-9999.ebuild:40]: Thanks for the reminder, I’ve removed this line.

**Pull request closed without merging**